rust-client/src/instructions/dynamic_amm/create_pool.rs

```diff
@@ -65,9 +70,9 @@ pub fn process_new_dynamic_pool(args: &Args, sub_args: &CreateDynamicAmmPoolArgs
         });
     }
 
-    let b_vault = derive_vault_address(token_b_mint);
-    let b_token_vault = derive_token_vault_address(&b_vault);
-    let mut b_vault_lp_mint = derive_vault_lp_mint_address(&b_vault);
+    let b_vault = derive_vault_key(*token_b_mint);
+    let b_token_vault = derive_token_vault_key(b_vault);
+    let mut b_vault_lp_mint = common::dynamic_vault::pda::derive_lp_mint_key(b_vault);
```

Comment:
Logic here is a bit weird, caused of legacy code, I think we can just hardcode some vault that has lp_mint is not PDA

Reply:
Fixed